.dockerignore

```diff
@@ -1 +1,3 @@
 frontend/node_modules
+current-bench-bechamel
+scripts
```

Comment:
why did you ignoring the scripts ? :)

Reply:
Because I wanted to relaunch the deployability testing and needed to update the commit to do so! :D The failure was not our fault, but a network issue on their end.
I believe this `.dockerignore` is only used by the CI to copy the whole repository to docker -- and we only use the `scripts/` directory for human convenience, so docker doesn't need it to build our images. It doesn't really matter given the size of that directory though!

Reply:
haha okay, that sounds reasonable :)

Reply:
It was a good question! I added the `.git` directory to the `.dockerignore` since it's a lot more expensive to copy :)